src/arcade/patch/agent/module/PatchModuleProliferation.java

```diff
@@ -87,7 +87,12 @@ public void step(MersenneTwisterFast random, Simulation sim) {
                     int newID = sim.getID();
                     CellContainer newContainer = cell.make(newID, State.UNDEFINED, random);
                     PatchCell newCell =
-                            (PatchCell) newContainer.convert(sim.getCellFactory(), newLocation);
+                            (PatchCell)
```

Comment:
My understanding of this implementation is that one of the daughter cells inherits the parents parameters, and then there is randomness/noise associated with the other daughter, should the daughter cells have the same parameters?